provisioning/device/src/ProductInfo.cs

```diff
@@ -16,12 +15,15 @@ public override string ToString()
         {
             const string Name = "Microsoft.Azure.Devices.Provisioning.Client";
 
-            string version = typeof(ProvisioningDeviceClient).GetTypeInfo().Assembly.GetName().Version.ToString(3);
+            // TODO: Replace with Assembly information.
+            // DO NOT EDIT the following line; it is updated by the bump_version script (https://github.com/Azure/iot-sdks-internals/blob/master/release/csharp/inputs.js)
+            const string Version = "1.0.2"; // CommonAssemblyVersion
+
             string runtime = RuntimeInformation.FrameworkDescription.Trim();
             string operatingSystem = RuntimeInformation.OSDescription.Trim();
             string processorArchitecture = RuntimeInformation.ProcessArchitecture.ToString().Trim();
 
-            string userAgent = $"{Name}/{version} ({runtime}; {operatingSystem}; {processorArchitecture})";
+            string userAgent = $"{Name}/{Version} ({runtime}; {operatingSystem}; {processorArchitecture})";
 
```

Comment:
Why do we need to change this? We should not touch useragent

Reply:
Please see the comments of Version on right above. It's from Sam, not sure what it is doing.